firmware/firmware.md

```diff
@@ -9802,7 +9802,8 @@ void loop() {
 
 The library provides the following log handlers:
 
-`SerialLogHandler`
+- `SerialLogHandler`
+- Additional community-supported log handlers can be found further below.
```

Comment:
Adding this comment here feels out of place, as a way to direct people elsewhere away from the official logging docs.  I don't think we should start adding non-official libraries in between official docs.  I see the benefit of this papertrail logger, but I think we need a separate Community section of the docs.  If it's unofficial it means we don't maintain it, and run the risk of these items being confusing to new users if they are not maintained well.  There are hundreds of libraries, and you can imagine how confusing it will be if we started adding them everywhere in the firmware reference.